tutorials/18_vote_on_content/index.py

```diff
@@ -0,0 +1,49 @@
+import steembase
+import steem
+from pick import pick
+
+# connect to testnet
+steembase.chains.known_chains['STEEM'] = {
+    'chain_id': '79276aea5d4877d9a25892eaa01b0adf019d3e5cb12a97478df3298ccdd01673',
+    'prefix': 'STX', 'steem_symbol': 'STEEM', 'sbd_symbol': 'SBD', 'vests_symbol': 'VESTS'
+}
+
+#capture user information
+username = input('Please enter your username: ')
+postingkey = input('Please enter your private posting key: ')
+
+#connect node and private posting key, demo account being used: cdemo, posting key: 5JEZ1EiUjFKfsKP32b15Y7jybjvHQPhnvCYZ9BW62H1LDUnMvHz
+s = steem.Steem(nodes=['https://testnet.steem.vc'], keys=[postingkey])
+
+#capture variables
+author = input('Author of post/comment that you wish to vote for: ')
+permlink = input('Permlink of the post/comment you wish to vote for: ')
+
+#check vote status
+result = s.get_active_votes(author, permlink)
+
+if result:
+	x=0
+	while x<(len(result)+1):
```

Comment:
The tutorial hangs here. 
There's no increment on `x`
And it looks like if there were, the while loop is perplexingly intended deliberately check a position past the end of the result list.

A simple *for ... in* would be easier to both write and understand ex: `for vote in result :`

What I don't get is this PR is for code doesn't work _at all_.